opentelemetry-jaeger/src/lib.rs

```diff
@@ -382,11 +382,15 @@ fn links_to_references(links: &sdk::EvictedQueue<api::Link>) -> Option<Vec<jaege
 fn build_tags(span_data: &Arc<trace::SpanData>) -> Option<Vec<jaeger::Tag>> {
     let mut tags = Vec::with_capacity(span_data.attributes.len() + span_data.resource.len() + 4);
     let mut user_specified_error = false;
+    let mut user_specified_span_kind = false;
     for (key, value) in span_data.attributes.iter() {
         tags.push(api::KeyValue::new(key.clone(), value.clone()).into());
         if key == &api::Key::new("error") {
             user_specified_error = true;
         }
+        if key == &api::Key::new("span.kind") {
```

Comment:
Then instead of checking many times here, the struct could have a method that accepts the key's value as a str

Reply:
If we do it this way does this mean creating a struct containing as fields the string values of the keys we want with default value `false`, then using the method to set the value `true` if the str matches one of our keys?